docs/en/interfaces/http.md

```diff
@@ -135,12 +135,11 @@ If you specified `decompress=1` in the URL, the server decompresses the same dat
 
 It is also possible to use other compression methods for HTTP data. To send a `POST` request compressed, append the request header `Content-Encoding: compression_method`. In order for ClickHouse to compress the response, you must append `Accept-Encoding: compression_method`. You can use the following compression methods:
 
-- Standard `gzip`-based [HTTP compression](https://en.wikipedia.org/wiki/HTTP_compression) (`compression_method = gzip`).
-
-    You should use the ClickHouse [enable_http_compression](../operations/settings/settings.md#settings-enable_http_compression) setting. You can configure the compression level of the data with the [http_zlib_compression_level](#settings-http_zlib_compression_level) setting.
-
+- Common gzip-based [HTTP compression](https://en.wikipedia.org/wiki/HTTP_compression) (`compression_method = gzip`).
```

Comment:
This wiki link still needs to be earlier, here should be just [gzip](https://en.wikipedia.org/wiki/Gzip)

Reply:
1. Here is a list of compression methods and this link is in right place here with all other links to other methods.